pcapng_exporter/include/pcapng_exporter/mapping.hpp

```diff
@@ -16,6 +16,7 @@ namespace pcapng_exporter {
 	struct channel_info {
 		std::optional<std::uint32_t> chl_id = std::nullopt;
 		std::optional<std::string> inf_name = std::nullopt;
+		std::optional<std::string> direction = std::nullopt;
```

Comment:
* Document valid values for direction
* Would an enum be better than a string here? for performance

Reply:
switched to an enum and documented possible string values in json file